exactonline/http.py

```diff
@@ -276,7 +264,29 @@ def _marshalled(data):
     return data
 
 
-def _http_request(url, method=None, data=None, opt=None):
+def _update_ratelimiter_with_exactonline_headers(limiter, headers):
+    if limiter:
+        # First Daily, then Minutely. If both keys are the same, the shortest
+        # should win. And the minute timer is likely the shortest.
+        if headers.get('x-ratelimit-reset'):
+            limiter.update(
+                # X-RateLimit-Reset: 1638489600000
+                until=headers.get('x-ratelimit-reset'),
+                # X-RateLimit-Limit: 9000
+                limit=headers.get('x-ratelimit-limit'),
+                # X-RateLimit-Remaining: 8924
+                remaining=headers.get('x-ratelimit-remaining'))
+        if headers.get('x-ratelimit-minutely-reset'):
+            limiter.update(
+                # X-RateLimit-Minutely-Reset: 1638447360000
+                until=headers.get('x-ratelimit-minutely-reset'),
+                # X-RateLimit-Minutely-Limit: 100
+                limit=headers.get('x-ratelimit-minutely-limit'),
+                # X-RateLimit-Minutely-Remaining: 99
+                remaining=headers.get('x-ratelimit-minutely-remaining'))
```

Comment:
```suggestion
    if not limiter:
        return

    # First Daily, then Minutely. If both keys are the same, the shortest
    # should win. And the minute timer is likely the shortest.
    if headers.get('x-ratelimit-reset'):
        limiter.update(
            # X-RateLimit-Reset: 1638489600000
            until=headers.get('x-ratelimit-reset'),
            # X-RateLimit-Limit: 9000
            limit=headers.get('x-ratelimit-limit'),
            # X-RateLimit-Remaining: 8924
            remaining=headers.get('x-ratelimit-remaining'))
    if headers.get('x-ratelimit-minutely-reset'):
        limiter.update(
            # X-RateLimit-Minutely-Reset: 1638447360000
            until=headers.get('x-ratelimit-minutely-reset'),
            # X-RateLimit-Minutely-Limit: 100
            limit=headers.get('x-ratelimit-minutely-limit'),
            # X-RateLimit-Minutely-Remaining: 99
            remaining=headers.get('x-ratelimit-minutely-remaining'))
```

can avoid the indentation